assets/javascripts/discourse-assign/routes/group-assigned.js.es6

```diff
@@ -22,5 +22,12 @@ export default DiscourseRoute.extend({
     } else {
       this.transitionTo("group.assigned.show", "everyone");
     }
+  },
+
+  actions: {
+    changeAssigned() {
+      console.log("route action");
+      this.refresh();
+    }
```

Comment:
This bit looks good. The only problem is that we already have a `changeAssigned` action in the `group-assigned-show` route. If you remove that, then I think the action will bubble all the way up to this `group-assigned` route, and the members will get refreshed correctly. (and I think the topic list will also be refreshed, since calling refresh() on `group-assigned` will also refresh child routes)

Reply:
😍😍 It worked perfectly fine.
But now the issue is total counts doesn't change after refreshing too